src/current/_includes/common/licensing-faqs.md

```diff
@@ -0,0 +1,148 @@
+CockroachDB is made available under the [CockroachDB Software License][csl].
+
+## Types of licenses
+
+The table of licenses below refers to options for {{ site.data.products.core }} deployments. All {{ site.data.products.cloud }} deployments automatically have a valid **Enterprise** license.
+
+Type | Description
+-------------|------------
+**Enterprise** <a name="enterprise"></a> | This paid license allows usage of all CockroachDB features with no restrictions under the terms specified in the [CockroachDB Software License][csl]. License must be renewed annually or as negotiated. Support levels available include [Enterprise][support] or [Essential][support].
+**Enterprise Free** <a name="enterprise-free"></a> | Same functionality as **Enterprise**, but free of charge for businesses with less than $10M in annual revenue, and telemetry is required except for ephemeral clusters (7 days or less). Clusters will be [throttled](#throttling) after 7 days without sending telemetry. License must be renewed annually. Support level available is Community (i.e., [Docs]({% link {{ page.version.version }}/index.md %}), [Forum][forum], [Slack][slack]).
+**Enterprise Trial** <a name="enterprise-trial"></a> | A 30 day self-service trial license. Telemetry is required during the trial. Telemetry can be disabled once the cluster is upgraded to a paid **Enterprise** license. Support level available during trials is Community (i.e., [Docs]({% link {{ page.version.version }}/index.md %}), [Forum][forum], [Slack][slack]).
+
+{{site.data.alerts.callout_success}}
+Note that:
+- Clusters with no license have a 7-day grace period before needing to install a license and start sending telemetry data.
+- No license key is required for developers running [single-node clusters](#single-node-clusters).
+- Clusters with Enterprise Free or Enterprise Trial licenses cannot disable telemetry; if such a cluster signals a telemetry sending error, it will be due to firewall configuration or a network issue.
+{{site.data.alerts.end}}
+
+## Obtain a license
+
+To obtain a paid **Enterprise** license, <a href="mailto:sales@cockroachlabs.com">contact Sales</a>.
+
+To obtain an **Enterprise Free** or **Enterprise Trial** license, take the following steps:
+
+1. Point your web browser to the [CockroachDB {{ site.data.products.cloud }} Console](https://cockroachlabs.cloud) and [log in]({% link cockroachcloud/authentication.md  %}#console-ui-authentication).
+1. Select **Organization &raquo; Enterprise Licenses** from the nav bar dropdown. This will bring you to the **Enterprise Licenses** page, which shows a (possibly empty) list of licenses and their keys, with information about the status of each.
```

Comment:
we should mention that you must have "Organization Admin" permissions

Reply:
updated the first step to 

"... log in as an account with **Organization Admin** permissions."